wger/exercises/api/serializers.py

```diff
@@ -29,6 +29,19 @@ class Meta:
         model = Exercise
 
 
+class ExerciseInfoSerializer(serializers.ModelSerializer):
+    '''
+    Exercise Info serializer
+    '''
+    image = serializers.StringRelatedField(source='get_this_image', many=True)
+
+    class Meta:
+        model = Exercise
+        fields = ('category', 'creation_date', 'description', 'language',
+                  'muscles', 'muscles_secondary', 'status', 'name',
+                  'equipment', 'license', 'license_author', 'image')
+
+
```

Comment:
There is a better and simpler way of doing this. Research on usage of `depth` in Django Serializers. With this all changes in the models are unnecessary and you may need to remove them

Reply:
@MuthomiMate I have made the relevant changes and used `depth`. I have also attached a screenshot on the PR message and wrote tests for the feature.